stan/math/prim/scal/prob/exponential_rng.hpp

```diff
@@ -1,38 +1,50 @@
 #ifndef STAN_MATH_PRIM_SCAL_PROB_EXPONENTIAL_RNG_HPP
 #define STAN_MATH_PRIM_SCAL_PROB_EXPONENTIAL_RNG_HPP
 
-#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
-#include <stan/math/prim/scal/err/check_nonnegative.hpp>
-#include <stan/math/prim/scal/err/check_not_nan.hpp>
 #include <stan/math/prim/scal/err/check_positive_finite.hpp>
-#include <stan/math/prim/scal/fun/value_of.hpp>
 #include <stan/math/prim/scal/meta/length.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
 #include <stan/math/prim/scal/meta/VectorBuilder.hpp>
-#include <stan/math/prim/scal/meta/return_type.hpp>
-#include <stan/math/prim/scal/fun/constants.hpp>
-#include <stan/math/prim/scal/meta/include_summand.hpp>
 #include <boost/random/exponential_distribution.hpp>
 #include <boost/random/variate_generator.hpp>
 
 namespace stan {
   namespace math {
-
-    template <class RNG>
-    inline double
-    exponential_rng(double beta,
-                    RNG& rng) {
+    /**
+     * Return a pseudorandom exponential variate with inverse scale beta
+     * using the specified random number generator.
+     *
+     * beta can be a scalar, a std::vector, an Eigen::Vector, or an
```

Comment:
"beta" is unanchored here---better to put this on the actual argument or in the top-level description.  Something like:

```
Return a pseudorandom exponential variate for each specified inverse scale using the specified pseudorandom number generator.
```

and then being more precise on the actual argument beta.  

```
@param beta scalar, standard vector, Eigen vector or row vector of inverse scales
```
I'd recommend either being imprecise and using "random" everywhere, or being precise and using "pseudorandom" everywhere.  Mixing the two seems to imply that the rng is really an RNG, whereas it's still a PRNG.

Reply:
I've switched to just saying random variate instead of psuedorandom variate. I'll update the language in the other rngs (that I haven't vectorized) as I get to them.